forge-gui/res/cardsfolder/rebalanced/a-deal_gone_bad.txt

```diff
@@ -0,0 +1,8 @@
+Name:A-Deal Gone Bad
+ManaCost:3 B
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ -3 | NumDef$ -3 | SubAbility$ DBMill | IsCurse$ True | SpellDescription$ Target creature gets -3/-3 until end of turn.
+SVar:DBMill:DB$ Mill | NumCards$ 3 | ValidTgts$ Player | TgtPrompt$ Choose a player | SubAbility$ DBGainLife | SpellDescription$ Target player mills three cards.
```

Comment:
I'd remove this TgtPrompt (also from non `A-` version)... might as well remove the one from line 4 too